shell/pages/c/_cluster/uiplugins/index.vue

```diff
@@ -488,46 +481,41 @@ export default {
   methods: {
     async refreshCharts(forceChartsUpdate = false) {
       // we might need to force the request, so that we know at all times if what's the status of the offical and partners repos (installed or not)
-      // tied to the SetupUIPlugins, AddExtensionRepos and RemoveUIPlugins checkboxes
+      // tied to the SetupUIPlugins, AddExtensionRepos checkboxes
       await this.$store.dispatch('catalog/load', { reset: true, force: forceChartsUpdate });
       const c = this.$store.getters['catalog/rawCharts'];
 
       this.charts = Object.values(c);
     },
 
     async updateInstallStatus(forceChartsUpdate = false) {
-      let hasService;
+      let hasFeatureFlag;
 
       try {
-        const service = await this.$store.dispatch('management/find', {
-          type: SERVICE,
-          id:   `${ UI_PLUGIN_NAMESPACE }/ui-plugin-operator`,
-          opt:  { force: true },
+        const flag = await this.$store.dispatch('management/find', {
+          type: MANAGEMENT.FEATURE,
+          id:   'uiextension'
```

Comment:
'uiextension' can be added alongside the other feature flags in `shell/store/features.js`

there's a convenience way to get them as well `this.$store.getters['features/get'](STEVE_CACHE);`